Amplify/Categories/DataStore/Model/Internal/Schema/ModelField+Association.swift

```diff
@@ -87,9 +87,9 @@ import Foundation
 /// - Warning: Although this has `public` access, it is intended for internal use and should not be used directly
 ///   by host applications. The behavior of this may change without warning.
 public enum ModelAssociation {
-    case hasMany(associatedWith: CodingKey?)
```

Comment:
Not adding deprecated warning here, because this enum is already marked with a warning that this is for internal only use.